src/pip/_internal/utils/misc.py

```diff
@@ -878,3 +879,13 @@ def hash_file(path, blocksize=1 << 20):
             length += len(block)
             h.update(block)
     return (h, length)  # type: ignore
+
+
+def interpreter_name():
```

Comment:
We should create an issue to replace this with `packaging.tags.interpreter_name()` once we pick up the as-yet released version.